### ISSUE TITLE
`traefik bug` output contains basic auth

### ISSUE THREAD

**Issue by maritz** [containous/traefik#2636]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Used the `traefik bug` command to get the json config.

### What did you expect to see?

The configuration as json but with all confidential information removed.

### What did you see instead?

In one of my entrypoints it included the actual htpasswd content like:

```json
{
   "Auth": {
    "Basic": {
     "Users": [
      "traefik-dashboard:$1234$aBcDeF"
     ],
     "UsersFile": ""
    },
    "Digest": null,
    "Forward": null,
    "HeaderField": ""
   }
}
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.5.0-rc3
Codename:     cancoillotte
Go version:   go1.9.2
Built:        2017-12-20_02:24:44PM
OS/Arch:      linux/amd64
```

**Comment by ldez** [comment]
We have some tests on this part [anonymize_config_test.go](https://github.com/containous/traefik/blob/master/cmd/traefik/anonymize/anonymize_config_test.go) and I cannot reproduce the pb with 1.5.0-rc3.

**Comment by ldez** [comment]
Finally I reproduce, I will fixed that immediately.

**Comment by traefiker** [comment]
Closed by #2638.
